cmd/minikube/cmd/config/enable.go

```diff
@@ -40,6 +40,11 @@ var addonsEnableCmd = &cobra.Command{
 		if len(args) != 1 {
 			exit.Message(reason.Usage, "usage: minikube addons enable ADDON_NAME")
 		}
+		if ClusterConfig, err := config.Load(ClusterFlagValue()); err == nil {
```

Comment:
```suggestion
		if clusterConfig, err := config.Load(ClusterFlagValue()); err == nil {
```
As per Go naming conventions